lib/block-supports/layout.php

```diff
@@ -573,14 +573,72 @@ function gutenberg_render_layout_support_flag( $block_content, $block ) {
 	$child_layout_styles       = array();
 
 	$self_stretch = isset( $block['attrs']['style']['layout']['selfStretch'] ) ? $block['attrs']['style']['layout']['selfStretch'] : null;
+	$self_align   = isset( $block['attrs']['style']['layout']['selfAlign'] ) ? $block['attrs']['style']['layout']['selfAlign'] : null;
```

Comment:
Is `$block['attrs']['style']['layout']['selfAlign']` the same as `$child_layout['selfAlign']`?

Reply:
Yes! I probably pasted this in from the prototype and forgot to make it pretty 😂